src/Components/UI/Calculator.js

```diff
@@ -1,47 +1,110 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prefer-stateless-function, react/destructuring-assignment, no-nested-ternary, max-len */
+// /* eslint-disable */
 import React from 'react';
+import calculate from '../Logic/calculate';
 import './Calculator.css';
 
 class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+      next: 0,
+      operation: null,
+    };
```

Comment:
- [ ] Upon loading the app for the first time, the calculator does not work. However when I click on th`AC` button it starts to work as expected, kindly fix this glitch in the code.